packages/validator/test/cli-validator/mock-files/oas3/testoneof.yaml

```diff
@@ -32,7 +32,7 @@ components:
           - description: 'foo type'
         - anyOf:
           - type: string
-            format: url
+            format: bad_format
```

Comment:
Changed this to an obviously invalid format value to preserve the negative test.